stdlib/public/runtime/SwiftRT-ELF.cpp

```diff
@@ -34,14 +34,32 @@ DECLARE_SWIFT_SECTION(swift5_fieldmd)
 DECLARE_SWIFT_SECTION(swift5_assocty)
 DECLARE_SWIFT_SECTION(swift5_replace)
 DECLARE_SWIFT_SECTION(swift5_replac2)
+DECLARE_SWIFT_SECTION(swift5_builtin)
+DECLARE_SWIFT_SECTION(swift5_capture)
 }
 
 #undef DECLARE_SWIFT_SECTION
 
 namespace {
-static swift::MetadataSections sections{};
+static swift::MetadataSections sections{ swift::CurrentSectionMetadataVersion };
 }
 
+__asm__ (
+    ".section \".note.swift_reflection_metadata\", \"aw\" \n"
```

Comment:
I don't think that the newline is needed, if you remove that, does this now fit on a single line?